xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -984,9 +981,6 @@ ENTRY bf16gemm {
 }
 
 TEST_P(ParameterizedGemmRewriteTest, Int8Gemm) {
-  if (CudaOrRocmCheck(Switch::False, Switch::True)) {
-    GTEST_SKIP() << "DoBlasGemmWithAlgorithm is not yet implemented on ROCm";
```

Comment:
For this test and the few below it: I don't believe the current PR makes any changes that implement this functionality. Should these checks remain in place for now?

Reply:
yes, sure, I added these checks back again